deploy/README.md

```diff
@@ -12,6 +12,10 @@ Current tasks included are:
     * Generate COG: Creates and writes COG to S3, pass granule ID and S3 location to Stac Generation task
     * STAC Generation: Creates STAC item from COG and posts to STAC database. Credentials are provided to the CDK workflow via environment variables. See `../stac-gen/README.txt` for more details.
 
+To provide dates associated with the files, you can follow the following convention for dates.
+* Date string/s in filename (following the yyyy-mm-dd, yyyy, yyyymm, yyyymmdd fromat): You can supply dates in any part of the filename followed by `_` in the formats `yyyy-mm-dd`, `yyyy`, `yyyymm`, `yyyymmdd`. Ideally, the dates will be towards the end of the filename. Eg: `1234_BeforeMaria_Stage0_2017-09-19_2017-07-21.tif` will extract start date as `2017-07-21` and end date as `2017-09-19` while disregarding `1234`.
+* `datetime_range`: If the `datetime_range` is not provided we just set the `datetime` field in the metadatda using the provided date. However, if `datetime_range` is provided (choice of `month` or `year`) we calculate the `start` and `end` date we need to ingest for the metadata. Eg: `1234_BeforeMaria_Stage0_2017.tif` is the filename, the date is set to `2017-01-01` if `datetime_range` is not provided. If it is set to `month`, `start_datetime` is set to `2017-01-01T00:00:00Z` and `end_datetime` is set to `2017-01-31T00:00:00Z` while also setting `date_time` as null. If `datetime_range` is set to `year`, `start_datetime` is set to `2017-01-01T00:00:00Z` and `end_datetime` is set to `2017-12-31T00:00:00Z` while also setting `date_time` as null.
```

Comment:
Where is datetime_range set? How would I set it?

Reply:
see here: https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/pull/138/files#r888379026